sympy/ntheory/tests/test_ecm.py

```diff
@@ -0,0 +1,6 @@
+from sympy.ntheory import primefactors, ecm
+
+def test_ecm():
+    assert set(primefactors(457873798931218796)) == ecm(457873798931218796)
```

Comment:
If you make the output of `ecm` sorted then you can drop the `set()` around `primefactors()`